app/views/widgets/single_column/_messaging_2_column_youtube_video.html

```diff
@@ -17,7 +17,18 @@ <h1 class="title">Sample Message With Photo</h1>
             Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus sit amet libero ultrices, aliquam ligula eu, sodales ligula. Ut maximus mauris tincidunt maximus hendrerit. Nullam velit felis, ultricies sit amet efficitur at, aliquet vitae velit. Phasellus dui magna, condimentum id fringilla sit amet, ultrices a magna. Ut semper, leo nec efficitur facilisis, nisi mi faucibus odio, ac ornare orci urna hendrerit
           </p>
         </div>
-        <div class="actions"><a class="theme-button">This is a button</a></div>
+        <div class="actions">
+          <a class="theme-button">This is a button</a>
+          <nav class="select">
+            <button class="select-button theme-border-color theme-text-color">List Options<span class="select-arrow theme-border-color"></span></button>
+            <ul class="select-list">
+              <li>
```

Comment:
Can you put lines 25-27 on one line, like line 28? I'm just kind of confused why they aren't formatted the same.